### PR TITLE
Fixing Clarion Hotwiring

### PR DESCRIPTION
**Pull request by MajesticBBQ**
## About the PR 
Updated Clarion to no longer have hotwired APCs prebuilt into the map at the engineering gas storage and SW maint, as well as add missing utility vendors in the now larger maint
image

## Why's this needed? 
Because the hotwiring was very dangerous to engineers in either the SW maint or Engineering Gas storage and needed fixing.

### CODE REVIEW

File: assets/maps/random_rooms/3x5/library.dmm [hunk 1]
```diff
@@ -0,0 +1,69 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```
Comment (TobleroneSwordfish) [inline assets/maps/random_rooms/3x5/library.dmm:1]: this file probably shouldn't be here

**Comment by amylizzle** [comment]
Probably best to split the random rooms from the clarion fix and pr them separately

**Comment by flrsh** [comment]
Yeah what Amy said. You should use separate branches so that your changes from other PRs don't show up.

**Review by ZeWaka — CHANGES_REQUESTED** [review]
x

**Comment by flrsh** [comment]
!merge_upstream

**Pull request merged**